### ISSUE TITLE
Teensy 3.5

### ISSUE THREAD

**Issue by TuomasHeikkila** [kiibohd/controller#243]
Teensy 3.5 doesn't seem to build quite right. It doesn't appear to be a supported version yet, are there any plans on getting it supported?

**Comment by haata** [comment]
Yeah, the Teensy 3.5 and 3.6 support is still preliminary. I still need to tune the clocks correctly.
